lib/src/cpp/utils/TagsXMLManager.hpp

```diff
@@ -47,7 +50,6 @@ constexpr const char* DEFAULT_EXTERNAL_UNICAST_LOCATOR_LIST = "default_external_
 constexpr const char* INITIAL_PEERS_LIST = "initialPeersList";
 constexpr const char* METATRAFFIC_EXTERNAL_UNICAST_LOCATOR_LIST = "metatraffic_external_unicast_locators";
 constexpr const char* NAME = "name";
-constexpr const char* TRANSPORT_ID = "transport_id";
```

Comment:
I think this was removed in the rebase... Also, there are some extra blank lines introduced in this PR that I think we removed in some other PR too.

Reply:
Yes because `TRANSPORT_ID = "transport_id"` has been defined twice: one due to participant user transports and the other one due to transport descriptor id (for kind implementation).
I decided to keep it below in the transport section. I would remove extra blank lines

Reply:
Sorry, I did not see that second definition.